gbpservice/nfp/orchestrator/modules/device_orchestrator.py

```diff
@@ -708,9 +906,22 @@ def _prepare_device_data(self, device_info):
         device['network_function_id'] = network_function_id
 
         device['advance_sharing_interfaces'] = (
-                    self._get_advance_sharing_interfaces(device['id']))
+            self._get_advance_sharing_interfaces(device['id']))
         return device
 
+    def health_monitor_complete(self, event, result='SUCCESS'):
+        nfp_context = event.data['nfp_context']
+        device = nfp_context['network_function_device']
+        network_function = nfp_context['network_function']
+
+        # Invoke event_complete for original event which is
+        # PERFORM_HEALTH_CHECK
+        event_desc = nfp_context.pop('event_desc')
+        event = self._controller.new_event(id="PEFORM_HEALTH_CHECK",
```

Comment:
Why we are creating event perform_health_check inside health_monitor_complete handler